doc/Command-Reference.md

```diff
@@ -541,6 +541,26 @@ This command displays the cause of the previous reboot
   User issued reboot command [User: admin, Time: Mon Mar 25 01:02:03 UTC 2019]
   ```
 
+**show reboot-cause history**
+
+This command displays the history of the previous reboots up to 10 entry
+
+- Usage:
+  ```
+  show reboot-cause history
+  ```
+
+- Example:
+  ```
+  admin@sonic:~$ show reboot-cause history
+  name                 cause        time                          user    comment
```

Comment:
Update headers to reflect change to capitalize the first letter of each

Reply:
I was working on it already :)